src/main/java/at/hannibal2/skyhanni/features/mining/fossilexcavator/FossilExcavatorAPI.kt

```diff
@@ -101,10 +101,14 @@ object FossilExcavatorAPI {
             inLoot = false
             return
         }
-
-        val pair = itemPattern.matchMatcher(message) {
-            ItemUtils.readItemAmount(group("item"))
-        } ?: return
+        // Workaround: If it is a enchanted book, we assume it is a paleontologist I book
+        val pair = if (message == "    §r§fEnchanted Book") {
```

Comment:
i think someone in the discord said they got more than one book in the same excavation, this wont catch that

Reply:
> crazy rare chance, also do we know how that chat message looks like? just the gray x1?

![image](https://github.com/hannibal002/SkyHanni/assets/69345714/404f96e1-e3af-4741-8441-fabc998982d8)